rules/linux/credential_access_collection_sensitive_files.toml

```diff
@@ -20,6 +20,37 @@ name = "Sensitive Files Compression"
 references = [
     "https://www.trendmicro.com/en_ca/research/20/l/teamtnt-now-deploying-ddos-capable-irc-bot-tntbotinger.html",
 ]
+note = """## Triage and analysis
+
+### Investigating Sensitive Files Compression
+
+The rule identifies the use of compression utility to collect known files containing sensitive information,
+such as credentials and other system configurations.
+
+#### Possible investigation steps
+
+- Identify the session entry leader and session user.
+- Examine the contents of session leading to the compression activity via the Session View.
+- Examine the command execution pattern in the session, which may lead to suspicious activities.
+- Examine the command execution post sensitive file compressions to indicate any exfiltration of sensitive file data.
+- Investigate any other alerts or malicious activities associated with the user/host during the past 48 hours.
+
+### False positive analysis
+
+- System administrators performing regular back up of sensitive files.
+- Whitelisted data backup and recovery tools executing in sensitive file paths.
```

Comment:
I think this is a TP too, like, directly specifying these as args shouldn't be common